bricks.module

```diff
@@ -155,7 +155,10 @@ function _bricks_tree_items($element, $items) {
  */
 function bricks_process_field(&$vars) {
   $element = &$vars['element'];
-  if ($element['#field_name'] == 'field_body') {
+
+  if ($element['#field_type'] == 'entityreference' &&
```

Comment:
Checking for `entityreference` type can be made inside `_bricks_field_is_treeable ` ;) can be a bit simpler.